src/server/hotspot_partition_calculator.cpp

```diff
@@ -42,6 +43,24 @@ DSN_DEFINE_int64("pegasus.collector",
                  "eliminate outdated historical "
                  "data");
 
+DSN_DEFINE_bool("pegasus.collector",
+                enable_hotkey_auto_detect,
+                false,
+                "auto detect hot key in the hot paritition");
+
+// if hot_partition_counter >= FLAGS_hotpartition_threshold, This partition is a hot partition
+DSN_DEFINE_int32("pegasus.collector",
```

Comment:
I think you can move these comments into the descripton param of `DSN_DEFINE_xxx`